rudder-web/src/main/scala/com/normation/rudder/web/rest/RestUtils.scala

```diff
@@ -94,7 +94,7 @@ object RestUtils extends Loggable {
    *   - Add a new line at the end and beginning of an array and indent one more level array data
    *   - space after colon
    */
-  def render(value: JValue): Document = {
+  def rendrrer(value: JValue): Document = {
```

Comment:
ah, it is an error, perhaps a focus that wasn't intended at that point. 
Strange that is didn't broke anything. Dead code ?